### DISCUSSION TITLE
How to overload same key to have different effect based on host/url?

### DISCUSSION THREAD

**Discussion by limaceous-bushwhacker** [gdh1995/vimium-c#1081] [Q&A]
Title. I'd like to map `yy` to copy different URL based on site. Currently I get error that `yy` has already been mapped if I use the following format:
```
env xx host="site.com"
map yy action sed="pattern"
```

If I change the host and env name in this, it says `yy` has already been mapped.

EDIT: Piggybacking off of this discussion, can we fire off two sed pattern on a URL? Something like `sed="0,1"`

**Comment by gdh1995** [comment]
`env` only names a environment which may be used by `runKey` - see https://github.com/gdh1995/vimium-c/wiki/Map-a-key-to-different-commands-on-different-websites :
```
env xx host="***"
map yy runKey expect="xx:yy1,xx2:yy2,xx3:yy3" default="yy0"
map yy0 command0
map yy1 command1
```

====

You may just write sed="01" to use both 0 and 1:
- a simple sed key's name should only have single character, so long values means to apply multiple sed keys in the same time
- if you want to give a substitution rule a longer name, then the name must:
  - start with the `_` (underscore) character
  - should not be longer than 8 characters
